app/controllers/index.js

```diff
@@ -1,9 +1,11 @@
 const helpers = require("./helpers");
+const create = require("./create");
 const update = require("./update");
 const remove = require("./remove");
 
 module.exports = {
-	helpers,
+    helpers,
```

Comment:
Hier is iets raar aan de hand met uw indenting.

Reply:
Ik denk dat dit een rariteit is van github:
![image](https://user-images.githubusercontent.com/6032796/71162583-c3bc2380-224b-11ea-8e78-9fa0834523e3.png)

Reply:
nvm. ik had de file niet gesaved 🙈